docs/_docs/configuration/options.md

```diff
@@ -171,6 +171,7 @@ class="flag">flags</code> (specified on the command-line) that control them.
         <p class="description">Enable auto-regeneration of the site when files are modified.</p>
       </td>
       <td class="align-center">
+        <p><code class="option">watch: BOOL</code></p>
```

Comment:
According to https://github.com/Zequez/jekyll/blob/master/lib/jekyll/commands/build.rb#L75 this should be `force_polling`

Reply:
Sorry, did you want me to edit something? `force_polling` is also set in this PR.